web3/_utils/module_testing/eth_module.py

```diff
@@ -294,6 +294,36 @@ async def test_eth_estimate_gas(
         assert is_integer(gas_estimate)
         assert gas_estimate > 0
 
+    @pytest.mark.asyncio
+    async def test_eth_feeHistory(self, async_w3: "Web3") -> None:
```

Comment:
@mike6649 just a small request to rename this to `test_eth_fee_history` to keep everything consistent